### PR TITLE
[Doc Improvement][Create new app using Teams Toolkit]

### PR DESCRIPTION
**Pull request by SPDoctor**
I have corrected a few grammatical errors, but also expanded some of the text to make it easier to read and understand (I hope), following the Microsoft style guide. Also the steps for viewing samples are slightly different depending on the context in VS Code, so I have added some text to reflect this.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a565eeb7190bfda7d4f97d299278be16a457ee37/msteams-platform/toolkit/create-new-project.md) | 95 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50aec8f0-b026-48aa-b13f-d5c3586524f8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/create-new-project.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Create new Teams project Using Teams Toolkit
+title: Create a new Teams project Using Teams Toolkit
```
Comment (v-leenagopal) [inline msteams-platform/toolkit/create-new-project.md:2]: Create a new Teams project using Teams Toolkit

**Review by v-leenagopal — CHANGES_REQUESTED** [review]
@surbhigupta12 I've added the review comments for this document-Create a new project

File: msteams-platform/toolkit/create-new-project.md [hunk 2]
```diff
@@ -8,30 +8,30 @@ ms.topic: overview
...
+## Create a new Teams app using sample code
 
- You can create new project by exploring **sample gallery**. Select the required sample to create apps. The following steps guide you to create apps from samples:
+You can also create a new project by exploring the **sample gallery** and choosing an existing sample as a starting point for your new app. These samples already have some functionality, for example a todo list with an Azure backend, or integration with the Microsoft Graph Toolkit. Follow these steps to create an app from one of the samples (if no folder is open in VS Code you'll skip step 2):
 
  1. Open **Teams Toolkit** from Visual Studio Code.
  1. Select **DEVELOPMENT** section in Tree View.
  1. Select **View samples**. The sample gallery appears as shown in the following image:
    
     :::image type="content" source="../assets/images/teams-toolkit-v2/manual/view samples.png" alt-text="samples":::
```
Comment (v-leenagopal) [inline msteams-platform/toolkit/create-new-project.md:32]: Image on sample gallery to be provided.

File: msteams-platform/toolkit/create-new-project.md (hunk 1 above)
Comment (v-leenagopal) [inline msteams-platform/toolkit/create-new-project.md:2]: Create a new Teams project using Teams Toolkit

File: msteams-platform/toolkit/create-new-project.md [hunk 3]
```diff
@@ -8,30 +8,30 @@ ms.topic: overview
 ms.date: 11/29/2021
 ---
 
-# Create new Teams project using Teams Toolkit
+# Create a new Teams app using Teams Toolkit
 
-You can create new Teams project using Teams Toolkit either by using **Create new project** or **Create from samples**.
+Once you have Teams Toolkit installed you have two options for creating a new app. You can either start from a basic starter project using the **Create a new Teams app** option, or you can adapt a sample app by using the **View samples** option.
```
Comment (v-leenagopal) [inline msteams-platform/toolkit/create-new-project.md:13]: You can create new Teams app after installing Teams Toolkit using either one of the options - Create a new Teams app or View samples option.

File: msteams-platform/toolkit/create-new-project.md [hunk 4]
```diff
@@ -8,30 +8,30 @@ ms.topic: overview
...
 
-## Create new project
+## Create a new Teams app using a new project
 
-The following guides helps you to create a new Teams app project using Teams Toolkit:
+You can choose **Create a new Teams app** and then select from a number of capabilities: Tab, Bot, Messaging Extension, or a Tab using the SharePoint Framework (SPFx). You must choose at least one, but you can add as many of these capabilities to your Teams app as you need. The following guides help you to scaffold a new Teams starter app using each of them:
```
Comment (v-leenagopal) [inline msteams-platform/toolkit/create-new-project.md:17]: Select Create a new Teams app and choose from the capabilities available - tab, bot, messaging extension, or a tab using SharePoint Framework (SPFx). Select at least one option to start creating and add more capabilities as per the requirement of the Teams app.

File: msteams-platform/toolkit/create-new-project.md [hunk 5]
```diff
@@ -8,30 +8,30 @@ ms.topic: overview
...
 
-- [Create new Teams Tab app (React)](/microsoftteams/platform/sbs-gs-javascript?tabs=vscode%2Cvsc%2Cviscode%2Cvcode&tutorial-step=2)
-- [Create new Teams Bot app](/microsoftteams/platform/sbs-gs-spfx?tabs=vscode%2Cviscode&branch)
-- [Create new Message Extension app](/microsoftteams/platform/sbs-gs-javascript?tabs=vscode%2Cvsc%2Cviscode%2Cvcode&tutorial-step=6&branch)
-- [Create new Teams Tab app (SharePoint Framework)](/microsoftteams/platform/sbs-gs-spfx?tabs=vscode%2Cviscode&branch)
+- [Create a new Teams Tab app (React)](/microsoftteams/platform/sbs-gs-javascript?tabs=vscode%2Cvsc%2Cviscode%2Cvcode&tutorial-step=2)
+- [Create a new Teams Bot app](/microsoftteams/platform/sbs-gs-spfx?tabs=vscode%2Cviscode&branch)
+- [Create a new Message Extension app](/microsoftteams/platform/sbs-gs-javascript?tabs=vscode%2Cvsc%2Cviscode%2Cvcode&tutorial-step=6&branch)
+- [Create a new Teams Tab app (SharePoint Framework)](/microsoftteams/platform/sbs-gs-spfx?tabs=vscode%2Cviscode&branch)
```
Comment (v-leenagopal) [inline msteams-platform/toolkit/create-new-project.md:22]: These are SBS guides and hence need to be written with the tags [!div class="nextstepaction"] in the Markdown file.

File: msteams-platform/toolkit/create-new-project.md [hunk 6]
```diff
@@ -8,30 +8,30 @@ ms.topic: overview
...
 
-## Create from samples
+## Create a new Teams app using sample code
 
- You can create new project by exploring **sample gallery**. Select the required sample to create apps. The following steps guide you to create apps from samples:
+You can also create a new project by exploring the **sample gallery** and choosing an existing sample as a starting point for your new app. These samples already have some functionality, for example a todo list with an Azure backend, or integration with the Microsoft Graph Toolkit. Follow these steps to create an app from one of the samples (if no folder is open in VS Code you'll skip step 2):
```
Comment (v-leenagopal) [inline msteams-platform/toolkit/create-new-project.md:26]: You can create a new project by exploring the View samples and choosing an existing sample for your new app. These samples may already have some functionality, for example a todo list with an Azure backend, or an integration with the Microsoft Graph Toolkit. Follow these steps to create an app from one of the samples. If there are no open folders in Visual Studio Code proceed with second step.

**Review by v-leenagopal — CHANGES_REQUESTED** [review]
@surbhigupta12, I've added the review comments for Create new project page under Tools and SDK module.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/59ef8a120652ce269c8871feab9b57846fb99e9f/msteams-platform/toolkit/create-new-project.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c63b42d7-751b-4910-afff-e0fa455b53e0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/create-new-project.md (hunk 3 above)
Comment (SPDoctor) [reply msteams-platform/toolkit/create-new-project.md]: How about:

You can create new Teams app after installing Teams Toolkit using one of two options: **Create a new Teams app**, or **View samples**. This allows you the choice of creating a basic starter project, or adapting one of a number of sample apps.

File: msteams-platform/toolkit/create-new-project.md (hunk 4 above)
Comment (SPDoctor) [reply msteams-platform/toolkit/create-new-project.md]: Changed slightly - please check.

File: msteams-platform/toolkit/create-new-project.md (hunk 2 above)
Comment (SPDoctor) [reply msteams-platform/toolkit/create-new-project.md:34]: I'm confused - the image has not changed from the original.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8152203ae3e867c8e8007ccb5678849d487d70c8/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d4c5613a-3db0-438a-9e13-478e2cd3a1ae) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/create-new-project.md (hunk 5 above)
Comment (SPDoctor) [reply msteams-platform/toolkit/create-new-project.md:24]: I don't see this style tagging in the original version or any other page. Can you give an example?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed15a35a4fea316f87fd2b773711e9c121baa9ee/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dc4e8982-4381-48f6-9bb2-cc7bb369f544) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2930a6f0e6162953c0a3243ca85dd2cee3e56914/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8cef6466-dd8d-49b3-9f9d-b076224cea86) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/create-new-project.md (hunk 6 above)
Comment (SPDoctor) [reply msteams-platform/toolkit/create-new-project.md]: Fixed

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6a74c0bfdcb7914d145f208eae105068d07e3229/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87246fdf-06d1-4d4e-81dc-dd6c7b049ec2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e7673b5d6fa46ae9dfe9f82e983d050d108b6d4/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/477ee84b-10aa-4e97-8cf6-4f5b075b6250) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cf39fd957bb4d1cdfaf32b3216183441c077f6c9/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aeb798ec-1f9d-4f41-a5b9-3378ac6de027) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bdf1e5180099cde6863e5b8517584fe54e84ae0a/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cf18409f-b4a0-45e5-b4e6-d0215eb5591c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1f05b3968f0d0dc34d7777834cff3e3877ce2cea/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/48bee18f-c8e3-45ae-899e-6ad8b45925f4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0c5cb7b544c1b79c1d556f343f4c5792f39b6f2a/msteams-platform/toolkit/create-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0737e0f5-be7a-4518-a090-e57e1b18dcc6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-leenagopal** [comment]
The details of all this PR has been updated in the PR 5155. Closing the PR.

**Pull request closed without merging**